i18n-helpers/src/lib.rs

```diff
@@ -160,6 +164,21 @@ pub enum Group<'a> {
     Skip(Vec<(usize, Event<'a>)>),
 }
 
+#[derive(Debug, Default)]
+struct GroupingContext {
```

Comment:
Please add a small docstring here so that people can see what this type is for.

Reply:
(The type was hidden inside a function before, which makes docstrings less necessary — but now that it's a top-level type, a docstring can be very useful.)